electron-app/package.json

```diff
@@ -64,9 +66,11 @@
     "megalodon": "^6.1.0",
     "nanoid": "^2.1.8",
     "nedb": "^1.8.0",
+    "node-fetch": "^2.6.12",
     "node-forge": "^0.10.0",
     "parse5": "^5.1.1",
     "postybirb-commons": "file:../commons",
+    "react-native-fs": "^2.20.0",
```

Comment:
Purpose?

Reply:
Works around something crazy in the Bluesky libs lol